service/gcs/bridge/bridge.go

```diff
@@ -372,11 +372,20 @@ func (b *bridge) resizeConsole(message []byte) (*prot.MessageResponseBase, error
 
 func (b *bridge) modifySettings(message []byte) (*prot.MessageResponseBase, error) {
 	response := newResponseBase()
+
+	var base prot.MessageBase
```

Comment:
Is it only for the purpose of getting the ActivityID out?
Is possible to collapse both Unmashall* into one?